packages/optimizer/lib/transformers/ReorderHeadTransformer.js

```diff
@@ -50,24 +50,24 @@ class HeadNodes {
   }
 
   _removeDuplicateCustomExtensions(extensions) {
-    const nodesByName = new Map();
+    const nodesBySrc = new Map();
     for (const node of extensions) {
-      const name = this._getName(node);
-      nodesByName.set(name, node);
+      const src = node.attribs['src'];
```

Comment:
Shouldn't this remove the host to make sure a same extension is not downloaded twice from two different hosts?